lesson28/src/js/password-done.js

```diff
@@ -0,0 +1,12 @@
+const checkUrlParams = () => {
+  const urlParams = new URLSearchParams(window.location.search);
+  const token = localStorage.getItem("updatePasswordToken");
```

Comment:
```suggestion
  const token = localStorage.getItem("forgetPasswordDone");
```

Reply:
Thank you very much.
Fixed.
[2782eb6](https://github.com/sae-github/JS_lesson/pull/32/commits/2782eb6a4cc70a21b5c1906b01a61edaf0ce67c6)